### PR TITLE
[6.2.z] cherry-pick katello change hostname stubs and tests

### PR DESCRIPTION
**Pull request by pondrejk**
katello-change-hosntname is pulled to 6.2, therefore cherry-picking the associated stubs as well as some negative tests. Results:

```
pytest tests/foreman/sys/test_rename.py -k test_negative
======================================================== test session starts ========================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.34, pluggy-0.4.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/pondrejk/Documents/robottelo, inifile:
plugins: xdist-1.20.0, services-1.1.14, forked-0.2, cov-2.3.1
collected 5 items 
2017-09-18 14:25:17 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/sys/test_rename.py ...

======================================================== 2 tests deselected =========================================================
============================================== 3 passed, 2 deselected in 11.21 seconds ==============================================
```

### CODE REVIEW

File: tests/foreman/sys/test_rename.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+        """Get hostname and credentials"""
+        super(RenameHostTestCase, cls).setUpClass()
+        cls.hostname = settings.server.hostname
+        cls.username = settings.server.admin_username
+        cls.password = settings.server.admin_password
+
+    @stubbed
```
Comment (san7ket) [inline tests/foreman/sys/test_rename.py]: why stubbed ?

File: tests/foreman/sys/test_rename.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            result = connection.run(
+                'katello-change-hostname -y \
+                        {0} -u {1} -p {2}'.format(
+                    self.hostname, self.username, password),
+                output_format='plain'
+            )
+            self.assertEqual(result.return_code, 1)
+            self.assertIn(BAD_CREDS_MSG, result.stderr)
+
+    @stubbed
```
Comment (san7ket) [inline tests/foreman/sys/test_rename.py]: Stubbed?

**Comment by san7ket** [comment]
@pondrejk Also, this is a new functionality added in z-stream , can you mark one of these test with upgrade decorator. whichever covers the positive scenario and most of the feature

File: tests/foreman/sys/test_rename.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/sys/test_rename.py]: Hello, it's the same story as in https://github.com/SatelliteQE/robottelo/pull/5051, the code down there is just a note for future developer, the point here is really to get those negative tests in

**Comment by pondrejk** [comment]
@san7ket decorator added :)

**Pull request merged**